src/drivers/pwm_out_sim/pwm_out_sim.cpp

```diff
@@ -512,6 +516,19 @@ PWMSim::task_main()
 				}
 			}
 
+			/* overwrite outputs in case of force_failsafe */
+			if (_failsafe) {
+				for (size_t i = 0; i < num_outputs; i++) {
+					outputs.output[i] = 600;
```

Comment:
I have moved 600 and 900 to named constants.
It doesn't make sense to read params as they are not configured for any simulator